src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/vnode_fusion.cpp

```diff
@@ -0,0 +1,262 @@
+    auto Constant_107279 = makeConst(ov::element::f32,
+                                     ov::Shape({
+                                         1,
+                                         1,
+                                         1,
+                                         1,
+                                     }),
+                                     {0.000000f});
+    auto eq_Equal_1 = makePattern<ov::opset1::Equal>({eq_Convert, Constant_107279},
+                                                     {{"auto_broadcast", "numpy"}});  //  tensor_array<u8[?,1,1,?]>
+    auto mul_LogicalAnd =
+        makePattern<ov::opset1::LogicalAnd>({eq_Equal, eq_Equal_1},
+                                            {{"auto_broadcast", "numpy"}});  //  tensor_array<u8[?,1,8192,?]>
+    auto masked_fill_Select =
+        makePattern<ov::opset1::Select>({mul_LogicalAnd, -FLT_MAX, causal_mask_boolean_1},
+                                        {{"auto_broadcast", "numpy"}});              //  tensor_array<f32[?,1,8192,?]>
+    auto copy__ShapeOf = makePattern<ov::opset1::ShapeOf>({causal_mask_boolean_1});  //  tensor_array<i32[4]>
+    auto Constant_47319 = makeConst(ov::element::u8, ov::Shape({}), 0);
+    auto copy__Broadcast =
+        makePattern<ov::opset1::Broadcast>({masked_fill_Select, copy__ShapeOf, Constant_47319},
+                                           {{"mode", "numpy"}});  //  tensor_array<f32[?,1,8192,..8192]>
+    auto SliceAssign_201_Reshape_2 =
+        makePattern<ov::opset1::Reshape>({copy__Broadcast, {-1}}, {{"special_zero", false}});  //  tensor_array<f32[?]>
+    auto SliceAssign_201_ScatterNDUpdate = makePattern<ov::opset4::ScatterNDUpdate>(
+        {SliceAssign_201_Reshape_0, SliceAssign_201_Reshape_1, SliceAssign_201_Reshape_2});  //  tensor_array<f32[?]>
+    auto SliceAssign_201_Reshape_3 =
+        makePattern<ov::opset1::Reshape>({SliceAssign_201_ScatterNDUpdate, {-1, 1, max_seq_len, max_seq_len}},
+                                         {{"special_zero", true}});  //  tensor_array<f32[?,1,8192,8192]>
+    auto ScatterUpdate_93554 =
+        makePattern<ov::opset3::ScatterUpdate>({{0, 0, 0, 0}, {3}, kvLen, {0}});  //  tensor_array<i32[4]>
+    auto slice_Slice_14 = makePattern<ov::opset1::StridedSlice>(
+        {SliceAssign_201_Reshape_3, {0, 0, 0, 0}, ScatterUpdate_93554, {1, 1, 1, 1}},
+        {{"begin_mask", {1, 1, 1, 0}},
+         {"end_mask", {1, 1, 1, 0}},
+         {"new_axis_mask", {}},
+         {"shrink_axis_mask", {}},
+         {"ellipsis_mask", {}}});  //  tensor_array<f32[?,1,8192,..8192]>
+    auto index_Gather = makePattern<ov::opset8::Gather>({slice_Slice_14, cache_positions, 2},
+                                                        {{"batch_dims", 0}},
+                                                        nullptr);  //  tensor_array<f32[?,1,?,..8192]>
+    auto result = index_Gather;
+
+    std::shared_ptr<ov::opset1::Constant> global_triu;
```

Comment:
The `global_triu` needs to be shared among different `AttentionBlock`. Could we make `global_triu` as a class member of transformation pass ?

Reply:
good idea, done!